.tekton/coo-fbc-v4-13-pull-request.yaml

```diff
@@ -175,11 +194,13 @@ spec:
         values:
         - "true"
       workspaces:
-      - name: output
-        workspace: workspace
       - name: basic-auth
-        workspace: git-auth
```

Comment:
same here, you're deleting too much, please do keep the worskapce: git-auth line

Reply:
done